src/cls/rgw/cls_rgw_ops.h

```diff
@@ -413,8 +413,7 @@ struct rgw_cls_list_op
 };
 WRITE_CLASS_ENCODER(rgw_cls_list_op)
 
-struct rgw_cls_list_ret
-{
+struct rgw_cls_list_ret {
```

Comment:
I don't care, but is this preferred?

Reply:
https://github.com/ceph/ceph/blob/master/CodingStyle cites https://google.github.io/styleguide/cppguide.html for C++ style. And in all their examples of class definitions, function definitions, and control structures, the opening brace is on the end of the line, not on a line of its own.

Higher up in this source file both styles are used for struct declarations. As I go through the code trying to understand I'll sometimes try to make things a bit more consistent and add comments that make it easier for me. (And I have my own biases.)